orca-plugins-test/src/test/resources/orca-plugins-test.yml

```diff
@@ -22,8 +22,13 @@ spring:
   application:
     name: orca
 
+dynamic-stage-resolver:
+  enabled: true
+
 spinnaker:
   extensibility:
+    framework:
+      version: v2
```

Comment:
I verified these tests pass on both v1 and v2, as well as verified they pass with both the `DynamicStageResolver` and `DefaultStageResolver`.